src/components/Table/index.tsx

```diff
@@ -0,0 +1,302 @@
+import * as React from 'react';
+import { TableHead } from './TableHead';
+import { TableToolbar } from './TableToolbar';
+import type { AutoUIEntityPropertyDefinition } from '../../AutoUI';
+import type { CheckedState, Pagination, TableSortOptions } from './utils';
+import { DEFAULT_ITEMS_PER_PAGE } from '../../AutoUI/utils';
+import { Material } from '@balena/ui-shared-components';
+import { Divider } from 'rendition';
+
+const {
+	Box,
+	Table: MaterialTable,
+	TableBody,
+	TableCell,
+	TableContainer,
+	TablePagination,
+	TableRow,
+	Paper,
+	Checkbox,
+	styled,
+} = Material;
+
+const StyledMaterialTable = styled(MaterialTable)(() => ({
+	'& [data-table="table_cel__sticky"]': {
+		position: 'sticky',
+		left: 0,
+		zIndex: 9,
```

Comment:
Not great that these zIndexes are arbitrary, but we can revisit them when we are looking at expanding our zIndexes

Reply:
when you work with sticky elements that goes in 2 directions there is no other real solution